core-library/src/main/java/org/silverpeas/core/admin/service/ComponentInstManager.java

```diff
@@ -20,6 +20,7 @@
  */
 package org.silverpeas.core.admin.service;
 
+import org.silverpeas.core.SilverpeasExceptionMessages;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.SilverpeasExceptionMessages'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)